services/api-server/tests/mocks/cleanup.py

```diff
@@ -7,27 +7,31 @@
 _fake = Faker()
 
 
-def obfuscate_values(json_key, json_data):
+def anonymize_values(json_key, json_data):
     if isinstance(json_data, dict):
         for key, value in json_data.items():
-            json_data[key] = obfuscate_values(key, value)
+            json_data[key] = anonymize_values(key, value)
     elif isinstance(json_data, list):
         for i in range(len(json_data)):
-            json_data[i] = obfuscate_values(i, json_data[i])
+            json_data[i] = anonymize_values(i, json_data[i])
     elif isinstance(json_data, str):
         if "@" in json_data:
+            print("\tAnonymizing email ...")
             json_data = _fake.email()
         elif json_key == "affiliation":
+            print(f"\tAnonymizing {json_key} ...")
             json_data = _fake.company()
         elif json_key == "name" and re.match(r"^[A-Z][a-z]+ +[A-Z][a-z]+$", json_data):
+            print("\tAnonymizing user names ...")
             json_data = f"{_fake.first_name()} {_fake.last_name()}"
```

Comment:
I see the point of this function, but it seems to me it might not be worth the effort? At least when I create the captures I usually use my own user and more or less dummy data. So I don't think there is any issues in exposing it in this repo.

Reply:
@bisgaard-itis so you want to commit your emails and passwords to github?  :-D